kms/asymmetric/samples.go

```diff
@@ -0,0 +1,189 @@
+package main
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"golang.org/x/net/context"
+	"google.golang.org/api/cloudkms/v1"
```

Comment:
separate vendored pkgs vs stdlib pkgs, see other samples for examples of this

Reply:
yes, generally, use [goimports](https://golang.org/x/tools/cmd/goimports) :)

Reply:
also... why aren't we using the gRPC client?

https://godoc.org/cloud.google.com/go/kms/apiv1

Reply:
I didn't realize "google.golang.org/api/cloudkms/v1" was deprecated, but "cloud.google.com/go/kms/apiv1" doesn't look like it has the new asymmetric keys functions. Do you know what's going on with this @white0ut ?

Reply:
Looks like the protobuf hasn't been published:

https://github.com/googleapis/googleapis/tree/master/google/cloud/kms/v1